### DISCUSSION TITLE
Bearded-Theme ( Possibility for Webstorm ? )

### DISCUSSION THREAD

**Discussion by CreatingBytes** [BeardedBear/bearded-theme#72] [Ideas]
Hey guys,

I wanted to ask if it's possible to make the plugin available for Jetbrains Webstorm as well. Some users find Webstorm a good alternative to vscode. But Jetbrains don't have such a good theme ...

For that i would also pay :D

Best regards

CreatingBytes

**Comment by BeardedBear** [comment]
I'm currently creating a port for the JetBrains IDE, but it takes me a lot of time, it's not at all as simple / ergonomic (to remain courteous) as on VSCode. So it's in progress, I've already made a lot of progress, but I still have a lot of finishing touches to do :).

**Reply by jk779** [reply]
Hey, I've just stumbled on this. I think your theme is the best coding theme out there. You're very talented. I'm using Jetbrains IDEs as well (Rubymine) and i would DIE for a ported version <3

They've just defaulted to their "new UI", so it would be a good moment to consider starting to port again 👼 🙈.

Just a little hopeful mini reminder :) It would make the expierience with JBs tools SO much better... 👉👈

**Reply by paolo-startselect** [reply]
Yes please! I hope this is still a work in progress. I agree this would make the experience with JB so much better, love your theme on vscode!

**Comment by BeardedBear** [comment]
Hi everyone! I'm currently in the middle of moving and there are a lot of things I have to manage in real life, everything should stabilize by September/October 🤞. I hope to have the time and energy to spend time on this portage, but it is very much in demand so it will take priority over other IDEs.

**Comment by jk779** [comment]
There is still this https://github.com/geowarin/bearded-jb/releases/tag/0.3 to get a little bit auf arcstorm, though i doubt it will work forever without updates ^^

**Comment by paolo-startselect** [comment]
Hey @BeardedBear how've you been? Were you able to find time and energy to work on this? 🙂
